packages/lab/src/progress/CircularProgress/CircularProgress.tsx

```diff
@@ -72,11 +48,6 @@ export interface CircularProgressProps extends HTMLAttributes<HTMLDivElement> {
    * If `true`, the info panel will be displayed.
    */
   showInfo?: boolean;
-  /**
-   * The size of the circle
-   * (small, medium, large)
-   */
-  size?: "small" | "medium" | "large";
```

Comment:
need to add a changeset eventually of removing size prop and 'static' from variant, etc

Reply:
will do, thanks for reviewing! :)